x-pack/plugins/security_solution/public/alerts/components/alerts_table/alerts_utility_bar/index.tsx

```diff
@@ -20,14 +23,14 @@ import * as i18n from './translations';
 import { useUiSetting$ } from '../../../../common/lib/kibana';
 import { TimelineNonEcsData } from '../../../../graphql/types';
 import { UpdateAlertsStatus } from '../types';
-import { FILTER_CLOSED, FILTER_OPEN } from '../alerts_filter_group';
+import { FILTER_CLOSED, FILTER_IN_PROGRESS, FILTER_OPEN } from '../alerts_filter_group';
 
 interface AlertsUtilityBarProps {
   canUserCRUD: boolean;
   hasIndexWrite: boolean;
   areEventsLoading: boolean;
   clearSelection: () => void;
-  isFilteredToOpen: boolean;
+  currentFilter: AlertStateStatus;
```

Comment:
Instead of using `AlertStateStatus` you should use `Status` which already has the correct types.